### ISSUE TITLE
How to merge "Where is your data?" and privacy app

### ISSUE THREAD

**Issue by MorrisJobke** [nextcloud/server#14738]
In #13097 there as a section added to the personal settings, that optionally (if set) shows where your data is. This looks like that and was developed by @weeman1337 - thanks for this 👍 For all of the values there is just a plain text field (except for the admin to choose from).

![Bildschirmfoto 2019-03-18 um 15 39 56](https://user-images.githubusercontent.com/245432/54538666-ea83bb80-4994-11e9-9d7c-261ecd08e6e1.png)

In parallel @georgehrke (asked for by @karlitschek) created a little app that shows beside this basic data also some text about privacy and the user data manifesto in a separate app, that looks like this. It automatically determines the admin and shows all users you shared with.

![Bildschirmfoto 2019-03-18 um 15 40 00](https://user-images.githubusercontent.com/245432/54538787-3171b100-4995-11e9-8cd9-68a989cfae72.png)
![Bildschirmfoto 2019-03-18 um 15 40 06](https://user-images.githubusercontent.com/245432/54538790-3171b100-4995-11e9-8496-b47c6365405c.png)

@nextcloud/designers Do you have a clever way of combining those two apps into one? Both have nice features that are helpful, but keeping both is too much. The plan was to ship the privacy app. Any ideas?

* pro "Where is your data?":
   * input for company website and T&C
   * selection for admin to contact (email address of that user is automatically linked)
   * configuration integrated in theming section
* pro "privacy app":
   * further text in a separate section that explains the background
   * automatic listing of admins and users you shared with

cc @rullzer

**Comment by weeman1337** [comment]
@georgehrke emailed me, that he is working on merging "Where is your data?" into the app.

Things I would like to see there is whether server side encryption is enabled and info about the provider. It could be alongside the map where it says "Your date is located in XY hosted by ZZ."

**Comment by jancborchardt** [comment]
@weeman1337 Sorry about the communication issues btw. :\ Hope we get a solution which is cool for everyone. :)

Few points:
- We should not say "my", but "your". "Where is your data".
- Needs proofreading … capital "YOUR" is a bit much, and the exclamation mark etc. (am on vacation atm so can only leave this drive-by comment)
- "User Data Manifesto" needs to be properly capitalized
- I’d put the map on the bottom of the page because it is so big. And put the country name above the map.
- What is the big + icon inside the "Who has access to your data?"
- Agree with @weeman1337 on the encryption app status
- Also @weeman1337’s work on adding privacy policy etc should be integrated.
- Then I’m fine moving it to a specific "Privacy" section and out of the "Profile info" completely.

**Comment by rullzer** [comment]
I'm fine with the encryption status. But we should somehow add more text (I'll try to think of something). Because it might also give a false sense of security.

**Comment by ChristophWurst** [comment]
Shall we revert https://github.com/nextcloud/server/pull/13097 then?

**Comment by rullzer** [comment]
> Shall we revert #13097 then?

guess so

**Comment by ChristophWurst** [comment]
> Also @weeman1337’s work on adding privacy policy etc should be integrated.

IMO we should just use the privacy policy URL from theming instead of duplicating that

**Comment by ChristophWurst** [comment]
It's ugly, but we can read the privacy URL from theming: https://github.com/nextcloud/server/blob/b8fcf6e9b3d492d19bbbdcd144428fef9bbf2324/lib/private/URLGenerator.php#L182-L189

**Comment by rullzer** [comment]
I moved them to separate issue in https://github.com/nextcloud/privacy

@ChristophWurst I'd be fine with a hacky way for now.

**Comment by ChristophWurst** [comment]
I think we can close this ticket here now that the server part was reverted and the privacy app contains most features. https://github.com/nextcloud/privacy/issues/68 is the only pending feature as far as I can tell.
